### DISCUSSION TITLE
Is output matrix can be input for eQTL?

### DISCUSSION THREAD

**Discussion by tewengtong** [campbio/decontX#29] [General]
Dear Author:

I am doing single-cell scale eQTL analysis. I wonder if it is okay to use the corrected count matrix by decontx() to be the phenotype of eQTL? Or I should use the count matrix before correction?

Thank you so much!

**Comment by joshua-d-campbell** [comment]
Hello @tewengtong. Thanks for your question. Yes, this matrix can be used for any down-stream analysis in the same way that you would use the original counts matrix. Note that this matrix is still "raw" counts and needs to be normalized in any clustering or eQTL workflow.

**Comment by tewengtong** [comment]
Thank you!
